css/src/mixins/checkbox.scss

```diff
@@ -0,0 +1,11 @@
+@mixin checkmark {
+	display: block;
+	position: absolute;
+	top: 1px;
+	width: calc($checkbox-icon-size / 2);
+	height: $checkbox-icon-size;
+	transform: rotate(45deg);
+	border: solid $secondary-hover;
```

Comment:
Is there a way we can use something other than secondary? That particular color isn't very well supported.

Reply:
Updated to $text-subtle